### ISSUE TITLE
Add in passenger stops.

### ISSUE THREAD

**Issue by ekapus** [ekapus/TrainCrew#4]

**Comment by philduba** [comment]
I'm thinking we need to do at least these two:
- Passenger Discharge
- Baggage handling

I could also see needing to add in links to switching operations for adding/dropping RPOs, REAs, etc. These should just be the common ones found in the switching page, perhaps just reusing them in this sections.

**Comment by ekapus** [comment]
I like it. 
I also think that passenger adds/drops are longer than freight ones due to steam/HEP connections. 
Hmm...

**Comment by philduba** [comment]
Yeah, completely forgot about steam/HEP connections. Probably not needed
for REAs but RPOs would probably need it.
